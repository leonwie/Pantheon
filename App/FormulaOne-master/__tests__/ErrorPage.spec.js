import React from 'react'
import ErrorPage from '../src/Components/ErrorPage'
import renderer from 'react-test-renderer'

describe('<ErrorPage />', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <ErrorPage />
    ).toJSON()

    expect(component).toMatchSnapshot()
  })
})
