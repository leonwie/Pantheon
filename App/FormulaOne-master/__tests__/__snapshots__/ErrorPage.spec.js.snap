exports[`<ErrorPage /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#313234",
      "flex": 1,
      "paddingTop": 20,
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "Raleway-Medium",
          "fontSize": 14,
          "lineHeight": 22,
          "textAlign": "center",
        },
        Object {
          "fontSize": 28,
        },
      ]
    }>
    Box! Box! Box!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "Raleway-Medium",
          "fontSize": 14,
          "lineHeight": 22,
          "textAlign": "center",
        },
        Object {
          "fontSize": 28,
        },
      ]
    }>
    Our pit crew changes something.
  </Text>
  <Image
    source={1}
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "position": "relative",
        "resizeMode": "contain",
        "width": null,
      }
    } />
</View>
`;
